Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -521,9 +521,10 @@ public override void StartDriver()
                         {
                             options.AddArguments("user-data-dir=" + UserProfileFolderPath);
                         }
-                        else if (!string.IsNullOrEmpty(ExtensionPath))
+                        else if (!string.IsNullOrEmpty(ExtensionPaths))
                         {
-                            options.AddExtension(Path.GetFullPath(ExtensionPath));
+                            string[] extensionPaths = ExtensionPaths.Split(';'); 
```

Comment:
Please verify if single extension path without ';' at the end will still work with this changes

Reply:
@kharithomas, we'll need to likely __prove__ with the addition of unit tests but please see the screenshots below:

Using 2 extensions:

`C:\Users\kharit\Desktop\ExtensionA.crx;C:\Users\kharit\Desktop\ExtensionB.crx`

![image](https://github.com/Ginger-Automation/Ginger/assets/55037984/0f067a41-e97f-4fb3-bb70-f79a9f97408a)

Using 1 extension:

`C:\Users\kharit\Desktop\ExtensionA.crx`

![image](https://github.com/Ginger-Automation/Ginger/assets/55037984/c5a944fd-c755-4b2c-8d9c-4a6e40e29f1e)